docs/source/platform-features/control/authentication-commands.rst

```diff
@@ -140,24 +148,31 @@ It can also take a regular expression representing a range of IP addresses.
     address []: /192.168.*/
 
 User_id:
----------
+--------
 User_id can be any arbitrary string that is used to identify the agent by the platform.
 If a regular expression is used for address or credential to combine agents in an authentication record then all
 those agents will be identified by this user_id. It is primarily used for identifying agents during logging.
 
+Identity:
+---------
+An identity is a string that represents the agent's VIP identity. This is an optional field, used by the platform to
+communicate between agents via RPC calls. It should be set if an agent has RPC exports.
+
+
 Capabilities:
 -------------
-Capability is an arbitrary string used by an agent to describe its exported RPC method. It is used to limit the access
-to that RPC method to only those agents who have that capailbity listed in their authentication record.
-
+A capability is an arbitrary string used by an agent to constrain its exported RPC method.
```

Comment:
I wouldn't say its arbitrary since it still has rules that it must follow.

Reply:
Removed 'arbitrary'